Formula/joe.rb

```diff
@@ -17,6 +17,6 @@ def install
   end
 
   test do
-    system "#{bin}/stringify"
+    system "test -x #{bin}/joe"
```

Comment:
I wish there was. There doesn't even appear to be a `--version` option supported. :-(